package-lock.json

```diff
@@ -1,14986 +0,0 @@
-{
-  "name": "elastic-apm-rum",
```

Comment:
Deleted by mistake?

Reply:
No! the package-lock.json was making the tests fail see: https://travis-ci.org/elastic/apm-agent-js-base/jobs/498791322#L508


I will add it back once the tests are green for the original changes. But I think we need to reconsider our use of package-lock.json

Reply:
seems like a cache issue to me.

Reply:
I also removed the cache, but it kept happening! It's a weird issue!